autopush/endpoint.py

```diff
@@ -774,11 +776,17 @@ def _put_message(self, kind, uaid, chid):
                 encryption=self.request.headers["encryption"],
             )
             data = urlsafe_b64encode(self.request.body)
-            # AWS cannot store empty strings, so we only add the encryption-key
+            # AWS cannot store empty strings, so we only add the key
             # if its present to avoid empty strings.
-            if "encryption-key" in self.request.headers:
-                headers["encryption_key"] = \
-                    self.request.headers["encryption-key"]
+            if ("encryption-key" in self.request.headers and
```

Comment:
My impression is that the `Topic` header (webpush-wg/webpush-protocol#9) replaces the `PUT` mechanism for collapsing messages. I'll file a separate issue for removing `_put_message` and friends.